handler/ws.go

```diff
@@ -319,17 +368,29 @@ func (p *PlayerConn) read() {
 		var payload Payload
 		err := p.conn.ReadJSON(&payload)
 		if err != nil {
+			// If the error is not a close error, then the player is kicked out.
+			// if !websocket.IsCloseError(err, websocket.CloseNormalClosure, websocket.CloseAbnormalClosure) {
+			// 	p.room.kickout(p)
+			// }
```

Comment:
Yesssss.. this part..

This was one of the places where I made mistake in the last code. I was trying to kickout from the player connection, and also kickout from the room connection. So, somewhere there was a lock or something